editor/src/tool/tools/select.rs

```diff
@@ -173,20 +173,31 @@ impl Fsm for SelectToolFsmState {
 						DrawingBox
 					};
 					buffer.into_iter().rev().for_each(|message| responses.push_front(message));
+
+					// ToDo: Clean up cloning
```

Comment:
Please change to `// TODO: ` for consistency with other usages. Specifically, because this breaks my VS Code extension which calls out TODO comments when they are formatted that way. Thanks.